### PR TITLE
Fixes `UIProgressView` height bug in iOS 14

### PR DESCRIPTION
**Pull request by simonmitchell**
## Description

Fixes bug caused by Apple's change to the system height of `UIProgressView` in iOS 14

## How Has This Been Tested?

Has been tested by pointing to this branch in GDPC First Aid

## Screenshots (if appropriate):

Screenshots are from iOS 14 and iOS 13.5 simulators :) 

Simulator Screen Shot - iPhone 11 Pro Max - 2020-09-23 at 16 30 19
Simulator Screen Shot - iPhone 11 Pro Max - 2020-09-23 at 16 29 36

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: ThunderCloud/QuizQuestionContainerViewController.swift [hunk 1]
```diff
@@ -335,18 +335,39 @@ open class QuizQuestionContainerViewController: AccessibilityRefreshingViewContr
...
+        progressStartCap.layer.cornerRadius = progressViewHeight/2
         progressStartCap.layer.masksToBounds = true
         progressStartCap.backgroundColor = progressView.progressTintColor
         progressContainer.addSubview(progressStartCap)
         
-        let progressEndCap = UIView(frame: CGRect(x: progressView.frame.maxX - 3, y: progressView.frame.minY, width: 6, height: 6))
-        progressEndCap.layer.cornerRadius = 3.0
+        let progressEndCap = UIView(
+            frame: CGRect(
+                x: progressView.frame.maxX - 4,
```
Comment (BenCShutt) [inline ThunderCloud/QuizQuestionContainerViewController.swift:364]: I know you've only cleaned up formatting but weird the min goes to the left and the max ALSO goes to the left by a different amount? Symmetry feels weird here

File: ThunderCloud/QuizQuestionContainerViewController.swift [hunk 2]
```diff
@@ -335,18 +335,39 @@ open class QuizQuestionContainerViewController: AccessibilityRefreshingViewContr
...
         progressView.set(minY: progressView.frame.minY + 10)
-        progressView.transform = progressView.transform.concatenating(CGAffineTransform(scaleX: 1.0, y: 3.0))
+        
+        let progressViewHeight: CGFloat = 6
+        // This is according to Stack Overflow the best way to give `UIProgressView` a custom height.
+        // we're not hard-coding the denominator here, as in iOS 14 Apple changed the standard height.
+        // frame may be set to 22 above, but something in `UIProgressView`'s init method, manually sets
+        // it to the correct system default value!
+        let yTransform = progressViewHeight/progressView.bounds.height
+        progressView.transform = progressView.transform.concatenating(CGAffineTransform(scaleX: 1.0, y: yTransform))
```
Comment (BenCShutt) [inline ThunderCloud/QuizQuestionContainerViewController.swift:345]: Not ideal but if it's all that seems to work!

**Review by BenCShutt — APPROVED** [review]

**Comment by BenCShutt** [comment]
Have we checked the other iOS versions ThunderCloud supports (might even go back to 11?)

File: ThunderCloud/QuizQuestionContainerViewController.swift (hunk 2 above)
Comment (simonmitchell) [reply ThunderCloud/QuizQuestionContainerViewController.swift:345]: Yeah it's a bit shitty :(

**Comment by simonmitchell** [comment]
I haven't no... but this code hasn't changed in the time I've been at cube (Other than being re-written in Swift by yours truly) so it should be fine :D

**Comment by BenCShutt** [comment]
Yeah it works basically the same so should indeed 😄

**Review by RodrigoLondon — APPROVED** [review]

**Pull request merged**